Realm/Realm/Extensions/CollectionExtensions.cs

```diff
@@ -365,6 +403,42 @@ public static IQueryable<T> Filter<T>(this IList<T> list, string predicate, para
             return realmList.GetFilteredResults(predicate, arguments);
         }
 
+        /// <summary>
+        /// Apply an NSPredicate-based filter over a collection. It can be used to create
+        /// more complex queries, that are currently unsupported by the LINQ provider and
+        /// supports SORT and DISTINCT clauses in addition to filtering.
+        /// </summary>
+        /// <typeparam name="T">The type of the objects that will be filtered.</typeparam>
+        /// <param name="set">A Realm Set.</param>
+        /// <param name="predicate">The predicate that will be applied.</param>
+        /// <param name="arguments">Values used for substitution in the predicate. Note that all primitive types are accepted as they are implicitly converted to RealmValue.</param>
+        /// <returns>A queryable observable collection of objects that match the predicate.</returns>
+        /// <remarks>
+        /// This method can be used in combination with LINQ filtering, but it is strongly recommended
+        /// to avoid combining it if a <c>SORT</c> clause appears in the predicate.
```

Comment:
This is no longer true and we should probably remove it from all the `Filter` docs.